### PR TITLE
Updating the Russian translation

### PR DESCRIPTION
**Pull request by stasyan68**
Updating the Russian translation and adding new lines

### CODE REVIEW

File: bbl/i18n/ru/OrcaSlicer_ru.po [hunk 1]
```diff
@@ -193,10 +193,10 @@ msgid "Scale"
 msgstr "Масштаб"
 
 msgid "Error: Please close all toolbar menus first"
-msgstr "Ошибка: пожалуйста, сначала закройте все меню панели инструментов"
+msgstr "Ошибка: пожалуйста, сначала закроть все меню панели инструментов"
```
Comment (vickas777) [inline bbl/i18n/ru/OrcaSlicer_ru.po:196]: Everything is ok with an original translation

**Comment by vickas777** [comment]
@SoftFever very poor translation, original has way less mistakes than in PR

**Comment by stasyan68** [comment]
> @SoftFever очень плохой перевод, в оригинале намного меньше ошибок, чем в PR

We are constantly testing and making edits. So the work continues!

**Comment by lizardjazz1** [comment]
Dear @vickas777, don't worry, the translation is improving every day for maximum users convenience

**Comment by SoftFever** [comment]
@vickas777

Feel free to comment in the code review or the conversation on Discord to discuss with the author and other Russian-speaking users. If needed, Ican create a sub Discord channel for specific discussions.

Translating 3D printing terms isn't always straightforward, and different opinions on translations can arise. I believe in open discussions and welcome pull requests or finding common ground when we disagree.

I would also like to mention that the original translation was contributed by @stasyan68.

**Comment by asychev** [comment]
This is a translation to whatever, but not a Russian.
Oриентируть, Введить, Нажмить - really? 😆

**Comment by lizardjazz1** [comment]
@asychev, be patient, wait for the current version to download, everything you pointed out has already been fixed.

**Review by SoftFever — APPROVED** [review]

**Comment by SoftFever** [comment]
Hello gentlemen,
I'll go ahead and merge this PR now. 
If there are any more discussions or matters to address, let's continue the conversation on https://discord.com/channels/1137181739773603922/1137213922454294528
We can make it better together 🥇

**Pull request merged**